### ISSUE TITLE
allow for multi-line name (sort of a collector for items known but without a specific smart plug for measurement)

### ISSUE THREAD

**Issue by commentsonly** [MindFreeze/ha-sankey-chart#106]
reminder for Petar as mentioned on the HA community forum

It would be could if one would have the ability for multi-line \<br>-alike names for the final entites which are in form "various" or "unknown".
Sort of a collector if one's aware about devices being part of the remaining unspecified energy consumption.
So that one could simply list the known devices in some sort of a list against building an endless 1-liner.

Alternative I could think of
Inventing a new -entity_id type, sort of 'unspecified' ... which allows for being connected being a child without a known consumption.
Doesn't work right now since that would require to have multiple 'remaining_parent_state' entities linked to the same pareent entity which can't work since. But this might be more complexed since then you need to know about how to split the remaining consumption which is surely mostly wrong if for example add a dishwasher and a small lightbulb.

In the end it would allow to put names to known consumers which would be great.

**Comment by derhappy** [comment]
Thanks for the implementation, this really helps with some of my names. However, it would be even cooler if the name could optionally go under the sensor value as well:
![grafik](https://user-images.githubusercontent.com/61394775/233028044-20912a24-a35e-4f5c-a043-8a1b7b8fc73e.png)

**Comment by MindFreeze** [comment]
This turned out to be quite a bit more difficult than it seemed, because label size is restricted and dynamic to avoid collisions.
You can even see in your example that your Netz entity doesn't have space for more lines and one entity in the first section is already unreadable even with 1 line.

Given all this I don't want to spend more time on this feature as it is difficult to implement and maintain, and the use case is not very common.

**Comment by commentsonly** [comment]
@MindFreeze 
I don't see a problem with collisions at all.
One should keep in mind that the min_box_hight must be at least number_of_name_lines * font-size + some pixels gap 😄 
Means the solution is already implemented right now.
Or did I overlook something vital?
I do like the new ability since it works for me.
I mostly tend to have an 'various' collection of things .. but when I realise there's 1 device with a heavy consumption I wanted to name that besides the 'various'

and
xx.x kWh - various
               - air heater bathroom
looks far better but
xx.x kWh - various & air heater bathroom

thanks a lot for the implementation in it's current form

**Comment by commentsonly** [comment]
@MindFreeze
nice implementation .... and I don't see collisions.
One must remember to make use of min_box_height (max number or lines * font-size) since so everything fine right now.

**Comment by MindFreeze** [comment]
Yes, min_box_height helps with this but it is a global option so it won't work for everyone. 
I don't want to complicate the labels any more, especially with options, so this is what we get for now.
There is also #109 which will complicate this issue way more.
